client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -348,7 +350,7 @@ def _get_filtered_repres(self, instance):
                 # to be published locally
                 continue
 
-            if "review" not in tags:
+            if "review" not in tags or "need_thumbnail" not in tags:
                 continue
```

Comment:
This would mean the representation **must** now have both otherwise it'd get skipped. is that correct?